src/components/navItem/NavItem.tsx

```diff
@@ -8,13 +8,19 @@ const NavItem = ({
   color,
   shouldDisplay,
 }: NavItemProps) => {
+  const RenderOnTop = {
+    position: "absolute",
+    top: "10px",
+    right: "0",
+  };
+
   return (
     <NavItemStyled
       to={linkTo}
       $backgroundColor={backgroundColor}
       $color={color}
       //temporary solution to render this element only on certain pages
-      className={`${shouldDisplay && "render-on-top"}`}
+      style={shouldDisplay && RenderOnTop}
```

Comment:
not like this please :)
make `shouldDisplay` a styled components prop and then apply those styles conditionally

Reply:
fixed!